<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pathCalibration</class>
 <widget class="QWidget" name="pathCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="gb_settings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Read delay (ms)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="ds_bandwidth">
        <property name="decimals">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="sb_average">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="pb_start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Average reads</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="ds_center_frequency">
        <property name="decimals">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_name"/>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="sb_delay">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Center Frequency (MHz)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Calibration Frequency (MHz)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="ds_cal_frequency">
        <property name="decimals">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bandwidth (MHz)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_calibration">
     <property name="title">
      <string>Calibration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pb_read_adc">
        <property name="text">
         <string>Read ADC</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cb_autoset_dbm">
          <property name="text">
           <string>Autoset dbm to plus</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="ds_dbm_auto_increment">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>of the previous measurement</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_read_phase">
        <property name="text">
         <string>read phase</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tw_pathCalibration">
        <column>
         <property name="text">
          <string>ADC</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>dbm</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>phase</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_delete_measurement">
        <property name="text">
         <string>Delete measurement</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pb_save">
          <property name="text">
           <string>Save and return</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
